samples/csharp/MicrosoftGraphSDK/Team.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph;
+
+namespace MicrosoftGraphSDK
+{
+    /// <summary>
+    /// Graph SDK endpoints for teams
+    /// </summary>
+    public static class Team
```

Comment:
Team->GraphServiceClientExtensions

Reply:
Could you please clarify if the class name needs to be updated to GraphServiceClientExtensions from Team. Thanks

Reply:
yes, that is the suggestion

Reply:
At the end all our methods for all our classes will be extension methods, can we keep the current classes names by entity type?

Reply:
please give an example to illustrate.

Reply:
@sharad-sharma-msft  the purpose of having those name for existing classes is to group all to methods which are related to and specific entity type. In this case the class was created for any endpoint touching a **Team** entity, here we can continue adding more methods as needed, for example to get Team members, etc.

If the class is renamed then that approach is going to be broken and probably, we will need to rename rest of classes due to all of them will contain extension methods.